### PR TITLE
cc: Update doc for Endpoints and Deploy API via APIM

### PR DESCRIPTION
**Pull request by SuKSW**
$subject
Fixes https://github.com/wso2/docs-apim/issues/3602

### CODE REVIEW

File: en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/concepts/apim-as-control-plane.md [hunk 1]
```diff
@@ -16,38 +16,50 @@ APIM control plane, therefore the user actions like API deploying, application c
 
     - This guide assumes that you have already started the WSO2 API Manager instance. If not, download the latest [release](https://github.com/wso2/product-apim/releases) and follow the steps [here](https://github.com/wso2/product-apim#installation--running).
 
-### Step 1 - Find the APIM IP Address
+### Step 1 - Find the API-M IP Address
 
-In order to tell Choreo Connect where API Manager (APIM) is located, find out the IP that can be used to access the API Manager instance. If you are trying out WSO2 API Manager locally, the private IP retrived using `hostname -I` or `ipconfig` would do.
+In order to tell Choreo Connect where API Manager (API-M) is located, find out the IP that can be used to access the API Manager instance. If you are trying out WSO2 API Manager locally, the private IP retrived using `hostname -I` or `ipconfig` would do.
```
Comment (nilminiwso2) [inline en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/concepts/apim-as-control-plane.md:21]:
```suggestion
Choreo Connect identifies the location of the API-M server instance by its IP address. If you are trying out WSO2 API Manager locally, you can use the private IP retrieved using `hostname -I` or `ipconfig`.
```

File: en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/concepts/apim-as-control-plane.md [hunk 2]
```diff
@@ -16,38 +16,50 @@ APIM control plane, therefore the user actions like API deploying, application c
...
 
 ### Step 2 - Update the Choreo Connect Config File
 
 Open the `<CHOREO-CONNECT_HOME>/docker-compose/choreo-connect/conf/config.toml` file in a text editor and update it as follows.
```
Comment (nilminiwso2) [inline en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/concepts/apim-as-control-plane.md:25]:
```suggestion
Open the `<CHOREO-CONNECT_HOME>/docker-compose/choreo-connect/conf/config.toml` file in a text editor and update the following configuration:
```

File: en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/concepts/apim-as-control-plane.md [hunk 3]
```diff
@@ -16,38 +16,50 @@ APIM control plane, therefore the user actions like API deploying, application c
...
+        i. update `environmentLabels` in `[controlPlane]` to `["choreo-connect"]`
+        ii. to be able to use accept APIs via apictl, include "choreo-connect" to [[adapter.vhostMapping]] as shown below.
+        ```
+        [[adapter.vhostMapping]]
+        environment = "choreo-connect"
+        vhost = "localhost"
+        ```
+
 
 ### Step 3 - Start the Choreo Connect
```
Comment (nilminiwso2) [inline en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/concepts/apim-as-control-plane.md:64]:
```suggestion
### Step 3 - Start Choreo Connect
```

File: en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/concepts/apim-as-control-plane.md (hunk 1 above)
Comment (SuKSW) [reply en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/concepts/apim-as-control-plane.md:21]: Updating this locally

File: en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/concepts/apim-as-control-plane.md (hunk 2 above)
Comment (SuKSW) [reply en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/concepts/apim-as-control-plane.md:25]: Updating this locally

File: en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/deploy-api/deploy-api-to-choreo-connect.md [hunk 4]
```diff
@@ -14,7 +14,59 @@ There are two ways to add an API to the Choreo Connect.
...
+        
+### Step 3 - Start Choreo Connect
+
+Now, let's start the Choreo Connect. Navigate to `CHOREO-CONNECT_HOME/docker-compose/choreo-connect` and execute the following command.
+    
+``` bash
+docker-compose up -d
+```
 
 ### Step 2 - Create an API in API Manager
```
Comment (samgnaniah) [inline en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/deploy-api/deploy-api-to-choreo-connect.md:71]:
```suggestion
### Step 4 - Create an API in API Manager
```

File: en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/deploy-api/deploy-api-to-choreo-connect.md [hunk 5]
```diff
@@ -31,6 +83,9 @@ During the startup, Choreo Connect will check the `config.toml` to see if the `c
 
 Whenever a new event occurs in API Manager such as an API being deployed, API Manager will notify Choreo Connect via Event Hub. Choreo Connect will then start fetching all the new artifacts related to its environment. 
 
+!!! Tip
+    To be able to invoke an API via the Devportal TryOut Console, make sure atleast one of the certificates used by the enforcer, is same as the certificate used by the Key Manager configured in API-M. In Choreo Connect, the certs for enforcer are located at `<CHOREO-CONNECT_HOME>/docker-compose/resources/enforcer/security/truststore`. In API-M, Key Managers can be configured from the API-M Admin Portal.
```
Comment (samgnaniah) [inline en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/deploy-api/deploy-api-to-choreo-connect.md:87]:
```suggestion
    To be able to invoke an API via the Devportal TryOut Console, make sure at least one of the certificates used by the enforcer is same as the certificate used by the Key Manager configured in API-M. In Choreo Connect, the certs for enforcer are located at `<CHOREO-CONNECT_HOME>/docker-compose/resources/enforcer/security/truststore`. In API-M, Key Managers can be configured from the API-M Admin Portal.
```

Comment (SuKSW) [reply en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/deploy-api/deploy-api-to-choreo-connect.md]: Added the change here

File: en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/deploy-api/deploy-api-to-choreo-connect.md (hunk 4 above)
Comment (SuKSW) [reply en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/deploy-api/deploy-api-to-choreo-connect.md]: Added the changes here

**Review by samgnaniah — APPROVED** [review]

File: en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/deploy-api/deploy-api-to-choreo-connect.md [hunk 6]
```diff
@@ -14,13 +14,65 @@ There are two ways to add an API to the Choreo Connect.
...
+
+    - This guide assumes that you have already started the WSO2 API Manager instance. If not, download the latest [release](https://github.com/wso2/product-apim/releases) and follow the steps [here](https://github.com/wso2/product-apim#installation--running).
+
+### Step 1 - Find the API-M IP Address
+
+Choreo Connect identifies the location of the API-M server instance by its IP address. If you are trying out WSO2 API Manager locally, you can use the private IP retrieved using `hostname -I` or `ipconfig`.
+
+### Step 2 - Update the Choreo Connect Config File
+
+Open the `<CHOREO-CONNECT_HOME>/docker-compose/choreo-connect/conf/config.toml` file in a text editor and update the `[controlPlane]` configuration as explained below:
```
Comment (Rajith90) [inline en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/deploy-api/deploy-api-to-choreo-connect.md:32]: we should use the choreo-connect-with-apim folder

**Comment by SuKSW** [comment]
Closing to resolve conflict

**Comment by SuKSW** [comment]
Closing to resolve conflict

File: en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/deploy-api/deploy-api-to-choreo-connect.md (hunk 6 above)
Comment (SuKSW) [reply en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/deploy-api/deploy-api-to-choreo-connect.md]: I added the link to the quick start guide of cc with apim docker compose.

**Pull request closed without merging**